index.css

```diff
@@ -0,0 +1,73 @@
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.contact-form {
+  max-width: 800px;
+}
+
+.contact-form .cf-item {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-size: 1.6rem;
+}
+
+.contact-form .cf-input {
+  padding: 12px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  margin-top: 6px;
+  margin-bottom: 16px;
+  resize: vertical;
+}
+
+.cf-functionality-wrap {
+  display: flex;
+  flex-wrap: wrap;
+  border-radius: 4px;
+  box-sizing: border-box;
+  border: 1px solid #ccc;
+  padding: 40px;
+  margin-bottom: 10px;
+}
+
+.cf-functionality-wrap .cf-item {
```

Comment:
It's rare that you need to target elements with two classes, and it makes maintaining code in the future a pain. Instead you could add two classes to the element — one for the styling that's common between all `cf-item` elements and another for the specific styling in `cf-functionality-wrap`.